auth/rhoas_auth.py

```diff
@@ -0,0 +1,17 @@
+from keycloak import KeycloakOpenID
+from os import environ;
+
+def get_access_token():
+    offlineToken = environ.get('OFFLINE_TOKEN')
+    # Configure client
+    keycloak_openid = KeycloakOpenID(server_url="https://sso.redhat.com/auth/",
```

Comment:
Those should be defined as constants. 
 get_access_token() should allow to override those values.